Development/styles.css

```diff
@@ -6,16 +11,16 @@ body
 
 #board-container
 {
-    width: 720px;
-    height: 720px;
+    width: calc(var(--square-side) * 9);
+    height: calc(var(--square-side) * 9);
```

Comment:
there is a reason I asked for percentages. Magic numbers are evil.

Reply:
what u mean

Reply:
what u mean

Reply:
elaborate